layouts/shortcodes/docs_index.html

```diff
@@ -12,7 +12,11 @@ <h1 id="{{ anchorize (i18n "subscriber_information") }}">{{ i18n "subscriber_inf
 
 <ul>
     <li>{{ template "link" (dict "context" . "page" "/docs/client-options") }}</li>
-    <li>{{ template "link" (dict "context" . "page" "/docs/rate-limits") }}</li>
+    <li>{{ template "link" (dict "context" . "page" "/docs/rate-limits") }}
+    <ul>
+        <li>{{ template "link" (dict "context" . "page" "/docs/duplicate-certificate-limit") }}</li>
+    </ul>
```

Comment:
More indentation to make it clear that this is nested under the main rate-limits bullet point.
```suggestion
        <ul>
            <li>{{ template "link" (dict "context" . "page" "/docs/duplicate-certificate-limit") }}</li>
        </ul>
```